### DISCUSSION TITLE
2023-03-22 - Weekly meeting agenda

### DISCUSSION THREAD

**Discussion by javiercasares** [WordPress/hosting-handbook#132] [Weekly Meetings]
### Meeting

/here <meeting type="hosting"> Who’s here to chat about hosting?

👋 Hello! This is the weekly meeting for the WordPress.org Hosting Team, who works with hosts and developers to help improve the WordPress user experience. This is a great time to ask any questions you may have, or visit Make/Hosting for more info. 

🥇 If you participate in the Five for the Future program, drop your WordPress.org username in a thread here 👉

🤗 If you're new here, welcome! You are welcome to introduce yourself if you like in a thread here 👉

-----

### Agenda

📓 ··· Here is the agenda for this meeting ···

**WordPress Community**
- WordPress 6.2 Releases
- Call for awareness of Rollback Update Failure and Plugin Dependencies

**Hosting Team Time**
- Hosting Community Team channel
- MariaDB Health Checks
- Advanced Administration Handbook
- Next events / Contributor Day

**Open Floor**

Would anyone like to add to or change any items on the agenda?

OK, let’s get started with the WordPress Community news.

-----

### WordPress Community

:wordpress: ··· WordPress 6.2 Releases ···

[WordPress 6.2 RC 2](https://wordpress.org/news/2023/03/wordpress-6-2-release-candidate-2/)
[WordPress 6.2 RC 3](https://wordpress.org/news/2023/03/wordpress-6-2-release-candidate-3/)

We have the WordPress 6.2 Release Candidate 3 available to test.

It's the final countdown: the third (and last) scheduled release candidate (RC3) for WordPress 6.2 is ready!

The WordPress 6.2 release is scheduled for March 28, 2023 —just one week away! Now is your last opportunity to test it before the general release.

:plugins: ··· Call for awareness of Rollback Update Failure and Plugin Dependencies ···

[Rollback Update Failure](https://wordpress.org/plugins/rollback-update-failure/)
[Plugin Dependencies](https://wordpress.org/plugins/wp-plugin-dependencies/)

Both plugins may be integrated in WordPress 6.3, so maybe it's a good time to check them and make suggestions.

-----

Ok, let’s move on to Hosting Team time.

### Hosting Team Time!

☁️ ··· Hosting Community Team channel ···

The Team Reps we've been talking about proposing changing the #hosting-community channel to just #hosting channel. Sounds good to you?

Please vote with a YES ❤️ or NO ☠️.

🦭 ··· MariaDB Health Checks ···

Last weekend war the Cloudfest Hackathon, and a lot of WordPressers were there. Some projects were about WordPress and one was [MariaDB Health Checks](https://github.com/itsdarkmars/hackathon).

The plugin should be functional, and it will be in the repo soon (as soon as the Plugins Team approve it).

I (@JavierCasares) want to thank personally Andrew Hutchings, Chief Contributions Officer for the MariaDB Foundation, leader of this project and, I hope soon, somebody I can convince being here from time to time to update us about MariaDB and the open-project.

📕 ··· Advanced Administration Handbook ···

[WordPress Advanced Administration Handbook](https://github.com/WordPress/Advanced-administration-handbook)

As you may know, Hosting Team is helping Documentation Team to document advanced documentation for helping users at the WordPress.org site.

Currently, [everything pending to do is in the Issues part](https://github.com/WordPress/Advanced-administration-handbook/issues). A lot of "documentation + enhancement" tickets, and also some "documentation + enhancement + good first issue" for the Contributor Day. Remember, there is a [page about the Contributor Day](https://make.wordpress.org/hosting/handbook/get-involved/contributor-day/) in the Handbook.

🗓️·····  Next events / Contributor Day ·····

Heads up, some events are coming down the pipeline along with some Contributor Days to participate in:

- 2023-03-31, [WordCamp Switzerland](https://switzerland.wordcamp.org/2023/), Murten (Switzerland)
- 2023-05-05, [WordCamp Barcelona](https://barcelona.wordcamp.org/2023/), Barcelona (Spain).
- 2023-06-08, [WordCamp Europe](https://europe.wordcamp.org/2023/), Athens (Greece).
- 2023-08-24, [WordCamp US](https://us.wordcamp.org/2023/), National Harbor, Maryland (USA).

WordCamp Europe Contributor Day: [Registration is now open!](https://europe.wordcamp.org/2023/contributor-day-registration-is-now-open/)

Is anyone [going to a WordCamp](https://central.wordcamp.org/schedule/) and wants to lead a Hosting Table? 

-----

### Open Floor

Ok, if nothing else, let’s move on to 🤖 ····· Open Floor ·····. Folks are welcome to bring any topic up for discussion at this time.

-----

### Next meeting

⏰ Here are the team’s upcoming meetings:

- 2023-03-22 at 18:00 UTC (in a few hours)
- 2023-03-29 at 09:00 UTC (next week)
- 2023-03-29 at 18:00 UTC

-----

📌 Okay, we’re out of time for today, let’s close out this meeting.
You are welcome to continue the conversation here in the #hosting-community channel or on Make/Hosting, at any time.
Thank you all for joining and for your contributions to WordPress!
Hope you have a wonderful rest of your week! </meeting>

**Comment by jessfrick** [comment]
We only had one Five for the Future participant this week in the late meeting: 

- pwtyler 

Also, only one vote during the meeting on the channel name change and it was a vote to keep the name as it is.
